images/win/scripts/Tests/Tools.Tests.ps1

```diff
@@ -14,4 +14,30 @@ Describe "Kind" {
     It "Kind" {
         "kind version" | Should -ReturnZeroExitCode
     }
+}
+
+Describe "WindowsNETFeatures" {
```

Comment:
May be put those functions to the separate `Windows.Tests.ps1`. I don't think it is related to Tools

Reply:
Yes, move these test to file `WindowsFeatures.Tests.ps1`